roles/addons/prometheus_server/molecule/default/molecule.yml

```diff
@@ -0,0 +1,57 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: management1
+    groups:
+      - mg_managements
+    children:
+      - equipment_typeM
+    image: "quay.io/actatux/ansible-${MOLECULE_DISTRO:-centos:8}"
+    override_command: false
+    capabilities:
+      - "SYS_ADMIN"
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true
+
+provisioner:
+  name: ansible
+  inventory:
+    host_vars:
+      management1:
+
+        start_services: true
+        enable_services: true 
+        j2_equipment_groups_list:
+          - equipment_typeM
+        ep_firewall: true
+
+        prometheus:
+          scrape_interval: 1m
+          evaluation_interval: 2m
+          alertmanager:
+            group_wait: 1m
+            group_interval: 10m
+            repeat_interval: 3h
+
+verifier:
+  name: ansible
+scenario:
+    test_sequence:
+    - dependency
+    - lint
+    - cleanup
+    - destroy
+    - syntax
+    - create
+    - prepare
+    - converge
+    #- idempotence
```

Comment:
Can you add a comment `# FIXME idempotence disabled because #366` above?

I wonder if we could only skip the idempotence for this specific task? You may have to modify the role itself and add the tag `molecule-idempotence-notest` to the task. (+ the comment above on why we skip it)
It would allow to run idempotence test for all other tasks, which is IMHO a better option.

Reply:
I prefer to keep the line commented until #366 is fixed.
The prometheus roles should be reworked.

Reply:
Can you consider the change below instead?

```
diff --git a/roles/addons/prometheus_server/molecule/default/molecule.yml b/roles/addons/prometheus_server/molecule/default/molecule.yml
index 8b2c23b..c4ab9f7 100644
--- a/roles/addons/prometheus_server/molecule/default/molecule.yml
+++ b/roles/addons/prometheus_server/molecule/default/molecule.yml
@@ -40,18 +40,3 @@ provisioner:

 verifier:
   name: ansible
-scenario:
-    test_sequence:
-    - dependency
-    - lint
-    - cleanup
-    - destroy
-    - syntax
-    - create
-    - prepare
-    - converge
-    #- idempotence # FIXME idempotence disabled because #366
-    - side_effect
-    - verify
-    - cleanup
-    - destroy
diff --git a/roles/addons/prometheus_server/tasks/main.yml b/roles/addons/prometheus_server/tasks/main.yml
index 3db4116..d392ebe 100644
--- a/roles/addons/prometheus_server/tasks/main.yml
+++ b/roles/addons/prometheus_server/tasks/main.yml
@@ -157,3 +157,4 @@
   when: (start_services | bool)
   tags:
     - service
+    - molecule-idempotence-notest # FIXME idempotence disabled because #366
```

This skips idempotence test on the failing task only. All other tasks will be tested, which provides a better coverage.

Reply:
Nice, I did not know the tag `molecule-idempotence-notest`
I will check it